processors/config/anaRecoTuple_cfg.py

```diff
@@ -39,6 +39,7 @@
 recoana.parameters["ecalHitColl"] = "RecoEcalHits"
 recoana.parameters["ecalClusColl"] = "RecoEcalClusters"
 recoana.parameters["histCfg"] = os.environ['HPSTR_BASE']+'/analysis/plotconfigs/reco/basicRecoHit.json'
+recoana.parameters["mcHistoCfg"] = os.environ['HPSTR_BASE']+'/analysis/plotconfigs/mc/basicMC.json'
```

Comment:
Looks like I missed one other thing, this also needs the new "analysis" parameter.